Makefile

```diff
@@ -43,7 +43,16 @@ test :
 # Docker build 
 ingress_build_dir := build/ingress
 template := ./nginx/nginx.tmpl
-git_rev := $(shell git rev-parse --short HEAD)
+ifdef REPOSITORY
+    repository := $(REPOSITORY)
+else 
+    repository := skycirrus
+endif
```

Comment:
You don't need ifdefs, just do:

```
REPOSITORY=skycirrus
```

Then use that everywhere. If someone then does `make REPOSITORY=mytest` it will override it.

Same with BUILD_VERSION below.